spec/models/miq_report_result/purging_spec.rb

```diff
@@ -111,15 +111,18 @@
       end
     end
 
+    # private method - not sure if we need
     it "#purge_ids_for_remaining" do
```

Comment:
This probably isn't necessary anymore if you are moving away from pulling back the ids and are using subqueries.